verifier/src/main/java/ch/admin/bag/covidcertificate/verifier/HomeFragment.kt

```diff
@@ -137,6 +157,12 @@ class HomeFragment : Fragment() {
 		setupInfoBox()
 	}
 
+	private fun showNewsDialog(configModel: ConfigModel) {
+		verifierSecureStorage.setNewsWasShown(configModel.getCovidCertificateNewsText("de") ?: "")
```

Comment:
To check if the news dialog was already shown (line 119) we use the `language_key` from the resources, but here a hardcoded language is used.